site/frontend/src/pages/status/page.vue

```diff
@@ -104,6 +105,23 @@ function getErrorsLength(errors: Dict<string>) {
   return `${errorsLen} ${errorsLen > 1 ? "s" : ""}`;
 }
 
+function ExpectedCurrentRequestCompletion() {
+  const req = data.value.timeline.find(isRequestInProgress);
+  if (!req) return "";
+  if (!req.endEstimated) return "";
+  if (!req.completedAt) return "";
+  const formattedDate = formatISODate(req.completedAt);
+  return (
+    <span>
+      Current Benchmark for{" "}
+      <strong>
+        <CommitSha tag={req.tag}></CommitSha>
+      </strong>{" "}
+      expected to end at {formattedDate}{" "}
```

Comment:
We already show the absolute duration in the timeline table, I thought that the added value of this would be to show relative time ("is expected to end in 5 minutes"). We already have code for this in the old status page, so I'd just copy-paste it from there.